File: model/src/main/java/com/sreMake/model/user/Role.java
package com.sreMake.model.user;

import com.sreMake.model.BaseEntity;
import org.babyfish.jimmer.sql.Entity;
import org.babyfish.jimmer.sql.Table;

@Entity
@Table(name = "srm_role")
public interface Role extends BaseEntity {

    String name();

}

File: model/src/main/kotlin/com/SReMake/model/user/Role.kt
package com.SReMake.model.user

import com.SReMake.model.BaseEntity
import org.babyfish.jimmer.sql.Entity
import org.babyfish.jimmer.sql.Table

@Entity
@Table(name = "srm_role")
interface Role : BaseEntity {
    val name: String
}